docs/device_api.md

```diff
@@ -167,8 +171,14 @@ Fiware-ServicePath: /
 }
 ```
 
-The example shows that the only difference between both provisionings is the presence or absence of an "endpoint" attribute.
-For devices that won't be receiving commands, the later provisioning request will be valid for both transport protocols.
+The example shows there are two differences comparing with provisioning for HTTP:
+
+* The absence of an "endpoint" field
+* The presence of "transport" field, which value has to be "MQTT".
+
+**Registering without endpoint or transport commands**
+
+For devices that won't be receiving commands, you don't need to specify neither "endpoint" nor "tranport" fields.
```

Comment:
I understand that this statement is correct, but I'd need somebody to confirm it is correct.

Reply:
That's correct

Reply:
That's correct